troubleshooting_central/backing-up-central-database-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,28 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="backing-up-central-database-by-using-the-roxctl-cli"]
+= Backing up Central database by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: backing-up-central-database-by-using-the-roxctl-cli
+
+toc::[]
+
+[role="_abstract"]
+Central stores information about:
+
+* Activity observed in your clusters.
+
+* Information retrieved from integrated image registries or Scanners.
+
+* {rh-rhacs-first} configuration.
```

Comment:
Since these would no longer be complete sentences w/o the lead-in:
```suggestion
* Activity observed in your clusters
* Information retrieved from integrated image registries or Scanners
* {rh-rhacs-first} configuration
```